user/token.go

```diff
@@ -69,9 +71,14 @@ func CreateSessionToken(data *TokenData, config TokenConfig) (*SessionToken, err
 	token.Claims["usr"] = data.UserId
 	token.Claims["dur"] = data.DurationSecs
 	token.Claims["exp"] = expiresAt
+	token.Claims["iss"] = "tidepool/shoreline"
+	token.Claims["sub"] = data.UserId
+	token.Claims["aud"] = "tidepool"
```

Comment:
Similar to above, I think it could be good if `aud` was environment specific. eg. `api.tidepool.org`

Reply:
Yessir.